service/src/main/java/bio/terra/policy/service/region/RegionService.java

```diff
@@ -26,47 +27,48 @@
 public class RegionService {
   private static final String TERRA_REGION_CONSTRAINT = "terra:region-constraint";
   private static final String TERRA_REGION_ATTRIBUTE_NAME = "region-name";
-  private static final String GLOBAL_REGION = "global";
+  private static final String GLOBAL = "global";
```

Comment:
nit: Can we use GLOBAL_LOCATION? Having a GLOBAL makes sense for the value but it makes me think too much - like is it a global variable? Why do we have GLOBALS, etc.